src/AppBundle/Utils/Markdown.php

```diff
@@ -23,15 +23,19 @@ class Markdown
 {
     private $parser;
 
+    private $purifier;
+
     public function __construct()
     {
         $this->parser = new \Parsedown();
+        $this->purifier = new \HTMLPurifier();
```

Comment:
I agree with you if we needed to purify other contents in the application. But for now, it doesn't look like we need it. So I prefer to not create that service.